CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ environment variables.
 `SENSU_TIMEOUT` variables to the command's environment.
 - Fixed a crash when running the backend on `darwin/arm64` when compressing a
 wrapped resource.
+- Fixed a bug where large number of silences could cause etcd errors by not
+exceeding etcd'd default maximum number of transaction operations.
```

Comment:
Should this read "by exceeding" instead of "by not exceeding"?

Reply:
No, although the sentence is ambiguous... I mean, by not exceeding the maximum ops per transaction, I've fixed a bug :)